Assets/Scripts/Scenes/InGame/Player/Magic/MagicScripts/GenerateWaterMagic.cs

```diff
@@ -44,7 +44,7 @@ private IEnumerator Magic()
                     Debug.Log("���������I");
                     //���ʔ���
                     //Todo�F���G�t�F�N�g����
-                    //_waterEffect = Instantiate(Resources.Load<GameObject>(���G�t�F�N�g�̃p�X��), myPlayerStatus.gameObject.transform.position, Quaternion.identity, myPlayerStatus.gameObject.transform);
+                    _waterEffect = Instantiate(Resources.Load<GameObject>("Prefab/Magic/Water"), myPlayerStatus.gameObject.transform.position, Quaternion.identity, myPlayerStatus.gameObject.transform);
```

Comment:
```suggestion
Vector3 spawnPos = GameObject.Find("Stage/Mid").transform.position + new Vector3(0, myPlayerStatus.gameObject.transform.position.y, 0);
_waterEffect = Instantiate(Resources.Load<GameObject>("Prefab/Magic/Water"), spawnPos, Quaternion.identity);
_waterEffect.transform.localScale = new Vector3(7.6f, 1, 7.6f);
```

すみません。以前のスクリプトに問題があったので変更をお願いします。

Reply:
アニメーションの一部をループに変更したのと、Scaleに関してはprefabのエフェクトの大きさを7.6倍にする形で修正しました。
[60dc381](https://github.com/InnovaGameCreate/mythos-encounter/pull/224/commits/60dc3814b795bc704347bf0707f999094f2108f4)